src/parser/core/modules/BuffWindow.tsx

```diff
@@ -38,7 +38,7 @@ export class BuffWindowState {
 	}
 }
 
-interface SeverityTiers {
+export interface SeverityTiers {
```

Comment:
Not a fan of AoE importing sevs from buffwindows, they should be stand-alone modules, esp. as they have no other dependency on each other.

Technically the correct solution is to add a type def for suggestions, but that's out of scope here. Cleaner to just add a dupe interface privately to your module.

Reply:
Duping the interface feels like it should be a last resort.  In other languages I work in, I'd just pull SeverityTiers into it's own file and import it into both BuffWindow and my new files.  Would that be suitable in ts here?

Reply:
ehhh... sorta. Like, yeah, it's technically the right solution, as part of a `.d.ts` file, but that then overrides inferred types from the JS file completely, so you'd need to type _all_ of suggestions, not just the interface. It's on The List:tm: to do.

If you really don't want to dupe it, can make a `SeverityTiers.ts`, but it'll play havok given the rest of the module is JS.